### PR TITLE
RFC-0341 - Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
## Description
Technical editing.

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal, if you get a chance, please could you review my changes. Thanks.

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# RFC-0341: Asset registration
-## Asset registration process
+# RFC-0341: Asset Registration
+## Asset Registration Process
 
 ![status: draft](theme/images/status-draft.svg)
 
 **Maintainer(s)**: Philip Robinson <philipr-za>
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
**Maintainer(s)**: [Philip Robinson](https://github.com/philipr-za)
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 2]
```diff
@@ -21,79 +21,104 @@ following conditions are met:
...
 
-    1. The output commitment is $ C = k.G + v.H $
-    2. $ v $ and $ k.G $ are provided in the metadata
-    3. A verifier can calculate $ C - k.G = v.H $ and verify this value by multiplying the clear text $ v $ by $ H $ themselves.
+The transaction will contain the following extra metadata to facilitate the registration process:
 
-2. A commitment (hash) to the asset parameters as defined by a [DigitalAssetTemplate] described in [RFC-0311](RFC-0311_AssetTemplates.md). This template will define all the parameters of the asset the AI intends to register including information the VNs need to know like what the required [AssetCollateral] is to be part of the committee.
+1. The value of the transaction in clear text and the public spending key of the commitment so that it can be verified 
+  by third parties. A third party can verify the value of the commitment by using the information in (1) and (2) below, to calculate (3):
+   1. The output commitment is $ C = k.G + v.H $.
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
   1. The output commitment is $ C = k \cdot G + v \cdot H $.
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 3]
```diff
@@ -21,79 +21,104 @@ following conditions are met:
...
+   2. $ v $ and $ k.G $ are provided in the metadata.
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
   2. $ v $ and $ k \cdot G $ are provided in the metadata.
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 4]
```diff
@@ -21,79 +21,104 @@ following conditions are met:
...
+   3. A verifier can calculate $ C - k.G = v.H $ and verify this value by multiplying the clear text $ v $ by $ H $
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
   3. A verifier can calculate $ C - k \cdot G = v \cdot H $ and verify this value by multiplying the clear text $ v $ by $ H $ 
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 5]
```diff
@@ -21,79 +21,105 @@ following conditions are met:
...
+The transaction will contain the following extra metadata to facilitate the registration process:
 
-### Asset commencement
-Once the Nomination phase is complete and the AI has selected a committee as described in [RFC-0304](RFC-0304_VNCommittees.md) the chosen committee and AI are ready to commit their `asset_creation_fee` and [AssetCollateral]s to commence the operation of the asset. This is done by the AI and the committee members collaborating to build the initial [Checkpoint] of the asset. When this [Checkpoint] transaction is published to the [base layer] the [digital asset] will be live on the DAN. The [Checkpoint] transaction is described in [RFC_0220](RFC-0220_AssetCheckpoints.md).
+1. The value of the transaction in clear text and the public spending key of the commitment so that it can be verified 
+   by third parties. A third party can verify the value of the commitment by using the information in (1) and (2) below, to calculate (3):
+   1. The output commitment is $ C
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
   1. The output commitment is $ C = k \cdot G + v \cdot H $.
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 6]
```diff
@@ -21,79 +21,105 @@ following conditions are met:
...
+      = k \cdot G + v \cdot H $.
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 7]
```diff
@@ -21,79 +21,105 @@ following conditions are met:
...
+   2. $ v
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
   2. $ v​ $ and $ k \cdot G ​$ are provided in the metadata.
```

File: RFC/src/RFC-0341_AssetRegistration.md [hunk 8]
```diff
@@ -21,79 +21,105 @@ following conditions are met:
...
+      ​$ and $ k \cdot G ​$ are provided in the metadata.
```
Comment (hansieodendaal) [inline RFC/src/RFC-0341_AssetRegistration.md]:
```suggestion
```

**Comment by anselld** [comment]
Hi @hansieodendaal , thanks, I've committed your changes.

**Comment by anselld** [comment]
Hi @hansieodendaal , I've merged the commits.

**Pull request merged**